### PR TITLE
Release/adapt 3200 campaign page visual updates

### PR DESCRIPTION
**Pull request by mattanglin**
# Release
https://stanfordits.atlassian.net/browse/ADAPT-3200

### CODE REVIEW

**Comment by mattanglin** [comment]
@sherakama Just realized that we'll need to revert the `dev` specific changes for deploying to production. We might wanna try and get #221 reviewed and merged into `dev` so we can tag it into this deploy. Otherwise I can just revert the dev related changes. Let me know!

**Comment by sherakama** [comment]
@mattanglin there is still work/thought to do on #22. Create a new branch for this PR and rebase it without the dev commits to go into main. This is not ideal I know but for now it is what it is.

**Comment by mattanglin** [comment]
> @mattanglin there is still work/thought to do on #22. ~Create a new branch for this PR and~ rebase this PR without the dev commits to go into main. This is not ideal I know but for now it is what it is.

@sherakama Rebased to remove the dev changes ✅

**Review by sherakama — APPROVED** [review]
Looks good to me. GTG.

**Comment by sherakama** [comment]
Thanks @mattanglin

**Comment by yvonnetangsu** [comment]
@sherakama @mattanglin This campaign page on live is rendering 2 lockups at the moment:
https://giving.stanford.edu/sykes-match/

File: src/components/identity/oodCampaignHeader.js [hunk 1]
```diff
@@ -1,28 +1,42 @@
...
+                <FullWidthImage
+                  filename={props.blok.logoImage.filename}
+                  blok={props.blok}
+                />
+              </SbLink>
+            </div>
+          ) : (
+            <CreateBloks blokSection={props.blok.lockup} />
+          )}
           <CreateBloks blokSection={props.blok.lockup} />
```
Comment (yvonnetangsu) [inline src/components/identity/oodCampaignHeader.js:24]: I think we need to delete this line 24

**Pull request merged**